plugins/org.komodo.core/src/org/komodo/repository/KSequencers.java

```diff
@@ -21,20 +21,23 @@
  */
 package org.komodo.repository;
```

Comment:
Can KSequencers be moved to the repository.internal package? It exposes a JCR class and I think it is only used by ModeShapeEngineThread.